release-notes/fleet-elastic-agent/known-issues.md

```diff
@@ -17,6 +17,38 @@ Known issues are significant defects or limitations that may impact your impleme
 
 % :::
 
+:::{dropdown} {{agents}} remain in an "Upgrade scheduled" state
+
+**Applies to: {{agent}} 8.18.0, 8.18.1, 8.18.2, 8.18.3, 8.18.4, 8.19.0, 9.0.0, 9.0.1, 9.0.2, 9.0.3, 9.1.0**
+
+On July 2, 2025, a known issue was discovered where {{agent}} remains in an `Upgrade scheduled` state when a scheduled {{agent}} upgrade is cancelled. Attempting to restart the upgrade on the UI returns an error: `The selected agent is not upgradeable: agent is already being upgraded.`.
```

Comment:
Don't think I've ever seen a discovery date in a known issue.  Is this something we're doing now? Does it help our users in anyway?

Reply:
Yes it is of help to us users and it has been revealed in other known issues as well.

As a user, I like to see this sot hat I have an idea on how long it's been known this an issue as this gives some sort of idea on how long before a fix may come along. The longer an issue as been known about, generally the sooner an impending fix may be available.